addons/website_event_exhibitor/models/website.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models
+
+
+class Website(models.Model):
+    _inherit = "website"
+
+    def _search_get_details(self, search_type, order, options):
+        result = super()._search_get_details(search_type, order, options)
+        if search_type in ['sponsor', 'all']:
```

Comment:
I'm not entirely sure how the "all" keyword functions.

Do we want it to show sponsors and tracks? Seems highly specific to the events app.

Reply:
![image](https://github.com/user-attachments/assets/50424b87-9034-4bbe-a42b-418f9cf8fd1b)

As far as i know in every app where we are using the  website search, we pass this so that we can search it in our model 
if we use all it means that we are searching for everything same thing has been done in the appointment app 
this sponsor or event is added so that we can search for it in our specific model